tornado_sqlalchemy/__init__.py

```diff
@@ -148,18 +126,180 @@ def _make_session(self) -> Session:
 set_max_workers = _async_exec.set_max_workers
 
 
-def make_session_factory(*args, **kwargs):
-    return SessionFactory(*args, **kwargs)
+class SessionEx(Session):
+    """The SessionEx extends the default session system with bind selection.
+    """
+
+    def __init__(self, db, autocommit=False, autoflush=True, **options):
+        self.app = db.get_app()
+        bind = options.pop('bind', None) or db.engine
```

Comment:
This could also be `bind = options.pop('bind', db.engine)`.

Reply:
there is a little difference between the two statement，if options is {'bind': None},  the result is different.

Actually, I copy this line from flask_sqlalchemy,  I am not sure whether it's intentional

Reply:
Good point. I'll resolve this one then.